### DISCUSSION TITLE
Non-duplicated footer

### DISCUSSION THREAD

**Discussion by bcalou** [gpbl/react-day-picker#1444] [Support] [answered]
Hello,
The [documentation shows](https://react-day-picker.js.org/guides/input-fields#example-range-selection) how to use the `footer` property to set a custom footer :
![image](https://user-images.githubusercontent.com/1257328/164689887-290e4d1f-050f-4346-bdff-8787337e2619.png)

However, if you combine this with `numberOfMonths={2}`, the footer is duplicated in each table : 
![image](https://user-images.githubusercontent.com/1257328/164691153-e43e5ab9-1d22-4875-8ca6-a7a847422321.png)

I think it would be very helpful to be able to set a footer that is unique to the whole component.

Thanks for this great date picker, one of the best I've tried.

**Comment by bcalou** [comment, marked as answer]
It's only after posting this that I realize that you could simply add whatever you want to the div containing the `DayPicker` component, outside of the `DayPicker` itself.
I'm using `react-popper` to make the picker appear and I somehow conviced myself that everything inside the popper was handled by `DayPicker`, which is false.

So, probably not the smartest suggestion ;)
(but I can't close it myself).

**Comment by gpbl** [comment]
@bcalou indeed the footer is applied to each calendar. To get only one element below, just add a wrapper around DayPicker:

```tsx
<div>
  <DayPicker />
  <form>
    <input name="from">     <input name="to">
  </form>
</div>
```

**Reply by bcalou** [reply]
Thanks for your answer !
